### DISCUSSION TITLE
Links not working in Wiki

### DISCUSSION THREAD

**Discussion by boan-jfm** [libvips/libvips#3652] [General]
The link to the "VIPS API docs" does not work (Broken link: http://www.vips.ecs.soton.ac.uk/supported/current/doc/html/libvips/index.html).

![image](https://github.com/libvips/libvips/assets/13201214/89a06114-2a04-43c0-bf85-314e40c46020)

and there is also an issue to the Documentation ("documentation doesn't have any public repositories yet.")

![image](https://github.com/libvips/libvips/assets/13201214/fe87e922-93c4-49df-9266-a0e422d6eb3e)

On this page: https://github.com/libvips/libvips/wiki/Libvips

**Comment by lovell** [comment]
Thanks for reporting, fixed these and a couple of other dead links I spotted.

https://github.com/libvips/libvips/wiki/Libvips/_compare/c065a00e07282e9c9afb9002ce3871854f5ff720...7c3c56ed59c242fecdddccd5d229b70861baeef2
